lib/block-supports/elements.php

```diff
@@ -115,7 +115,8 @@ function gutenberg_render_elements_support_footer( $pre_render, $block ) {
 	$style = ".$class_name a{" . $link_color_declaration . ';}';
 
 	gutenberg_enqueue_block_support_styles( $style );
-	return $style;
+
+	return null;
```

Comment:
I did that for the sake of the test...

Reply:
I removed the test I suggested we should have. As you said, in this case, the value would lie in a e2e test. It'd be great to have it :)